rebasehelper/helpers/bugzilla_helper.py

```diff
@@ -0,0 +1,155 @@
+
+import logging
+import os
+import re
+from typing import cast, Optional, Tuple
+
+import git  # type: ignore
+import requests
+
+from rebasehelper.exceptions import RebaseHelperError
+from rebasehelper.logger import CustomLogger
+
+
+logger: CustomLogger = cast(CustomLogger, logging.getLogger(__name__))
+
+
+class BugzillaHelper:
+
+    """Class for working with Upstream Release Monitoring on bugzilla."""
+
+    DIST_GIT_REPO_URL = 'https://src.fedoraproject.org/rpms'
+    BUGZILLA_REST_API_URL = 'https://bugzilla.redhat.com/rest'
+    UPSTREAM_RELEASE_MONITORING_USERNAME = 'upstream-release-monitoring'
+
+    @classmethod
+    def get_bugzilla_component(cls, bugzilla_id: str) -> str:
+        """Gets a component of the bugzilla.
+
+        Args:
+            bugzilla_id: ID of the bugzilla.
+
+        Returns:
+            Component of the bugzilla.
+
+        Raises:
+            RebaseHelperError: If no such bugzilla exists or if the bugzilla
+            is not created by Upstream Release Monitoring.
+
+        """
+        r = requests.get('{}/bug/{}'.format(cls.BUGZILLA_REST_API_URL, bugzilla_id))
+        if not r.ok:
+            raise RebaseHelperError('Could not obtain data from bugzilla')
+
+        response_json = r.json()
+        if 'error' in response_json:
+            logger.error('Bugzilla error: %s', response_json['error'])
+            raise RebaseHelperError('Could not obtain data from bugzilla')
+
+        bug = response_json['bugs'][0]
+        if bug['creator_detail']['email'] != cls.UPSTREAM_RELEASE_MONITORING_USERNAME:
+            raise RebaseHelperError('The given bugzilla was not created by Upstream Release Monitoring')
+        return bug['component'][0]
+
+    @classmethod
+    def get_version_from_comments(cls, bugzilla_id: str) -> Optional[str]:
+        """Gets version from bugzilla comments.
+
+        Args:
+            bugzilla_id: ID of the bugzilla.
+
+        Returns:
+            Version specified by Upstream Release Monitoring in comments
+            or None, if no version could be found.
+
+        Raises:
+            RebaseHelperError: If no such bugzilla exists.
+
+        """
+        r = requests.get('{}/bug/{}/comment'.format(cls.BUGZILLA_REST_API_URL, bugzilla_id))
+        if not r.ok:
+            raise RebaseHelperError('Could not obtain data from bugzilla')
+        version = None
+        comments = r.json()['bugs'][bugzilla_id]['comments']
+        pattern = re.compile(r'^Latest upstream release: (?P<version>.*)\n')
+        for comment in comments:
+            if comment['creator'] != cls.UPSTREAM_RELEASE_MONITORING_USERNAME:
+                continue
+            match = pattern.match(comment['text'])
+            if match:
+                version = match.group('version')
+
+        return version
+
+    @classmethod
+    def clone_repository(cls, component: str, bugzilla_id: str) -> str:
+        """Clones the component package repository
```

Comment:
This is rather vague and doesn't say much. How about `Clones remote dist-git repository of a component.`?